pages.ja/common/7zr.md

```diff
@@ -1,17 +1,17 @@
 # 7zr
 
-> 7-Zipは高圧縮率のファイルアーカイバ
-> .7zファイルをアーカイブタイプをサポートする
-> 詳しくはこちら: <https://www.7-zip.org>.
+> 圧縮率の高いファイルアーカイバ
```

Comment:
I don't speak the language. What about the period here?
```suggestion
> 圧縮率の高いファイルアーカイバ。
```

Reply:
Thank you for the review! After checking other Japanese translations, I noticed that I should use the period in the end of description. So, I've added it!